### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by ericherman**
## Version 0.2.0

October 26th 2020: 🎊 the sixth draft splits a requirement and adds clarity.

* Split "Welcome Contributions" criterion into "Make contributing easy" and "Welcome contributors".
* Rename criterion "Pay attention to codebase maturity" to "Document codebase maturity".
* Changed MUST to SHOULD for requirement of codebase in use by multiple parties.
* Add MUST NOT requirement regarding copyright assignment.
* Clarify role of configuration in reusable code requirement.
* Glossary additions: Continuous Integration, Policy, Repository, and Version control.
* Replace references to "cities" with "public organizations".
* Clarify aspects of sensitive code by separating contributor and reviewer requirements into separate items.
* Expand further reading, and guidance to policy makers, developers and designers.
* Add Felix Faassen and Arnout Engelen to authors.
* Made additional minor changes to text for clarity.

-----
[View rendered AUTHORS.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/AUTHORS.md)
[View rendered CHANGELOG.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/CHANGELOG.md)
[View rendered GOVERNANCE.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/GOVERNANCE.md)
[View rendered README.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/README.md)
[View rendered criteria/bundle-policy-and-code.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/bundle-policy-and-code.md)
[View rendered criteria/code-in-the-open.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/code-in-the-open.md)
[View rendered criteria/continuous-integration.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/continuous-integration.md)
[View rendered criteria/document-maturity.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/document-maturity.md)
[View rendered criteria/document-objectives.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/document-objectives.md)
[View rendered criteria/documenting.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/documenting.md)
[View rendered criteria/make-contributing-easy.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/make-contributing-easy.md)
[View rendered criteria/open-licenses.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/open-licenses.md)
[View rendered criteria/open-standards.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/open-standards.md)
[View rendered criteria/open-to-contributions.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/open-to-contributions.md)
[View rendered criteria/require-review.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/require-review.md)
[View rendered criteria/reusable-and-portable-codebases.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/reusable-and-portable-codebases.md)
[View rendered criteria/style.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/style.md)
[View rendered criteria/understandable-english-first.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/understandable-english-first.md)
[View rendered criteria/version-control-and-history.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/criteria/version-control-and-history.md)
[View rendered glossary.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/glossary.md)
[View rendered index.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/index.md)
[View rendered introduction.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/introduction.md)
[View rendered readers-guide.md](https://github.com/publiccodenet/standard/blob/release-0.2.0/readers-guide.md)

### CODE REVIEW

**Comment by bvhme** [comment]
Awesome! I'm quite excited for this.Expect a review from me tomorrow.

**Review by Ainali — APPROVED** [review]
Looks fine to me, but I didn't review the script changes.

File: readers-guide.md
Comment (bvhme) [inline readers-guide.md:4]: * Singular of criteria is criterion. So either we do 'Each criterion has' or 'All criteria have'
* Inconsistent new line behaviour.

```suggestion
The Standard describes a number of criteria.
All criteria have consistent sections that make it clear how to create great public code.
Below is a brief explanation of each of these sections and how they are used within the criteria of the Standard.
```

File: introduction.md
Comment (bvhme) [inline introduction.md:155]: Headings should be surrounded by whitespace.

```suggestion
## Further reading

- ["Modernising Public Infrastructure with Free Software" whitepaper](https://download.fsfe.org/campaigns/pmpc/PMPC-Modernising-with-Free-Software.pdf) by the Free Software Foundation Europe
```

File: criteria/open-to-contributions.md
Comment (bvhme) [inline criteria/open-to-contributions.md:19]: I spot a hyphen that should be an en dash.

```suggestion
* Prevents the community that works on a project splitting because there is no way to influence codebase goals and progress – resulting in diverging communities.
```

**Comment by bvhme** [comment]
I've generated a PDF on the current file as an indication 
[standard-0.2.0-rc1 (DO NOT USE)](https://github.com/publiccodenet/standard/files/5443993/standard.pdf)

File: criteria/document-objectives.md
Comment (bvhme) [inline criteria/document-objectives.md:32]: Is the second sentence not a separate point?
```suggestion
* Add the policy objectives to the codebase documentation, for example in the `README`.
* Include relevant polices which impact the community, codebase, and development like value and ethics based polices, for example accessibility or equal opportunity.
```

**Review by bvhme — CHANGES_REQUESTED** [review]
Awesome :)

Things to check:

* For some reason if I look under 'Files changed' I see that `script/pdf.sh` is an empty file. When I look on the branch or check it out locally it isn't. Could someone validate there is not something wrong here.
* Have we had any people contribute we should've really added as authors? Apart from @felixfaassen.

I would also like to ask @ElenaFdR to have a look at this pull request as I've already fished some textual things out of there and I'm curious as to what we've missed.

**Comment by bvhme** [comment]
Pull request #372 can serve as an intermediate step for the cover page.

**Comment by ericherman** [comment]
>     * For some reason if I look under 'Files changed' I see that `script/pdf.sh` is an empty file. When I look on the branch or check it out locally it isn't. Could someone validate there is not something wrong here.

It looks fine to me:

~/src/publiccodenet-standard$ git branch | grep '^\*'
* release-0.2.0
~/src/publiccodenet-standard$ ls -l script/pdf.sh 
-rwxr-xr-x 1 eric eric 57 Sep 28 11:41 script/pdf.sh
~/src/publiccodenet-standard$ git annotate script/pdf.sh
9bbe737f        (Boris VAN HOYTEMA      2019-10-08 16:18:13 +0200       1)weasyprint http://localhost:4000/print.html standard.pdf
~/src/publiccodenet-standard$


>     * Have we had any people contribute we should've really added as authors? Apart from @felixfaassen.

Maybe: @raboof caught a spelling error, ("specfic" -> "specific"), which @Ainali merged in https://github.com/publiccodenet/standard/commit/da17ce79eb5f14df6049401f99aa56a89250c8dc .... should we add Arnout Engelen to AUTHORS.md ?

**Comment by raboof** [comment]
> should we add Arnout Engelen to AUTHORS.md ?

I'd be happy to be listed but won't be offended if I'm not - it was *very* trivial, after all :smile:

**Comment by ericherman** [comment]
> > should we add Arnout Engelen to AUTHORS.md ?
> 
> I'd be happy to be listed but won't be offended if I'm not - it was _very_ trivial, after all smile

Should I add Arnout Engelen under "Individual contributors" or under an organization?

**Comment by raboof** [comment]
Individual fits best, I think.

October 27, 2020 11:42 AM, "Eric Herman" <notifications@github.com (mailto:notifications@github.com?to=%22Eric%20Herman%22%20<notifications@github.com>)> wrote:
	should we add Arnout Engelen to AUTHORS.md ? 

	I'd be happy to be listed but won't be offended if I'm not - it was very trivial, after all smile  

	Should I add Arnout Engelen under "Individual contributors" or under an organization?

**Comment by ericherman** [comment]
> Awesome :)
> 
> Things to check:
> 
>     * For some reason if I look under 'Files changed' I see that `script/pdf.sh` is an empty file. When I look on the branch or check it out locally it isn't. Could someone validate there is not something wrong here.
> 
>     * Have we had any people contribute we should've really added as authors? Apart from @felixfaassen.
> 
> 
> I would also like to ask @ElenaFdR to have a look at this pull request as I've already fished some textual things out of there and I'm curious as to what we've missed.



> > ```
> > * For some reason if I look under 'Files changed' I see that `script/pdf.sh` is an empty file. When I look on the branch or check it out locally it isn't. Could someone validate there is not something wrong here.
> > ```
> 
> It looks fine to me:
> 
> ~/src/publiccodenet-standard$ git branch | grep '^*'
> 
>     * release-0.2.0
>       ~/src/publiccodenet-standard$ ls -l script/pdf.sh
>       -rwxr-xr-x 1 eric eric 57 Sep 28 11:41 script/pdf.sh
>       ~/src/publiccodenet-standard$ git annotate script/pdf.sh
>       [9bbe737](https://github.com/publiccodenet/standard/commit/9bbe737f8b716d71eaabe8082a42f21dfc45230f)        (Boris VAN HOYTEMA      2019-10-08 16:18:13 +0200       1)weasyprint http://localhost:4000/print.html standard.pdf
>       ~/src/publiccodenet-standard$
> 
> 
> > ```
> > * Have we had any people contribute we should've really added as authors? Apart from @felixfaassen.
> > ```
> 
> Maybe: @raboof caught a spelling error, ("specfic" -> "specific"), which @Ainali merged in [da17ce7](https://github.com/publiccodenet/standard/commit/da17ce79eb5f14df6049401f99aa56a89250c8dc) .... should we add Arnout Engelen to AUTHORS.md ?



>     * For some reason if I look under 'Files changed' I see that `script/pdf.sh` is an empty file. When I look on the branch or check it out locally it isn't. Could someone validate there is not something wrong here.

Oh, I bet GitHub's web UI displays "empty file" when the change consists _only_ of an executable bit permission change.

**Review by bvhme — APPROVED** [review]
:ship: Looks good to merge!

**Review by Ainali — APPROVED** [review]

**Review by felixfaassen — APPROVED** [review]
I have nothing to add. This is a solid and nice release.

**Comment by ElenaFdR** [comment]
For ease, I've [created a new branch](https://github.com/publiccodenet/standard/tree/editing-release-0.2.0) for my edits; I'll do a PR against this when I'm finished. Please don't merge this PR until then.

**Comment by ElenaFdR** [comment]
The release instructions in the README say that we prep a new print versioin each time we do a release. I don't see a decision not to do that.
This is the README checklist:

1. Create a release branch
2. Update the new release
    - [ ] Update version number
    - [ ] Update release log
    - [ ] Create new cover for print
    - [ ] Create new cover for web PDF
    - [ ] Generate new PDF of the document
    - [ ] Create PDF with Illustrations pasted in
    - [ ] Create PDF for GitHub release with front and back cover included
3. Send the files for print to the printer
    - [ ] Cover file
    - [ ] Inside pages PDF
4. Create GitHub release with the release notes and version number

**Comment by ericherman** [comment]
> The release instructions in the README say that we prep a new print versioin each time we do a release. I don't see a decision not to do that.

We will be doing a print version.

> This is the README checklist:

This will be expanded in future releases and moved from README to CONTRIBUTING:
 - [Improve Release Instructions]( https://github.com/publiccodenet/standard/pull/365 )

**Comment by bvhme** [comment]
These PDFs come straight from Figma for now, please check if they are correct. I've pasted the latest text from the README.

I usually use something like PDFsam to add the front and back page to the generated PDF. The front and back PDF is for the print.

[Cover back and front(7).pdf](https://github.com/publiccodenet/standard/files/5452701/Cover.back.and.front.7.pdf)
[Back(4).pdf](https://github.com/publiccodenet/standard/files/5452702/Back.4.pdf)
[Front(5).pdf](https://github.com/publiccodenet/standard/files/5452703/Front.5.pdf)

**Comment by ericherman** [comment]
> These PDFs come straight from Figma for now, please check if they are correct.

These say "August 23rd 2020"

**Comment by bvhme** [comment]
O dang, on it

**Comment by bvhme** [comment]
I've updated them to the date of the Release notes.

[Cover back and front(8).pdf](https://github.com/publiccodenet/standard/files/5457570/Cover.back.and.front.8.pdf)
[Back(5).pdf](https://github.com/publiccodenet/standard/files/5457571/Back.5.pdf)
[Front(6).pdf](https://github.com/publiccodenet/standard/files/5457572/Front.6.pdf)

**Review by Ainali — APPROVED** [review]
Ready to go!

**Review by bvhme — APPROVED** [review]
Great! Love the public policy. All in all a solid improvement.

**Review by felixfaassen — APPROVED** [review]
Go time! Great release.

**Comment by AlbaRoza** [comment]
Excited to see it up and running! 🎉

**Review by AlbaRoza — APPROVED** [review]
[PR](https://github.com/publiccodenet/standard/pull/399) submitted with several changes.

**Review by ElenaFdR — APPROVED** [review]
Shiny!

**Comment by ericherman** [comment]
@AlbaRoza all of these things are nice to have, but do not look to me like release-stopping changes and I don't want to ask everyone to do another review unless we've found a release-halting change. Thus I've reverted your change-set.

Tomorrow, let's get them to done in the "develop" branch so they'll land in 0.2.1

**Comment by ericherman** [comment]
@bvhme  tag 0.2.0 puhsed

```
$ git tag 0.2.0
$ git push --tags
Total 0 (delta 0), reused 0 (delta 0)
To github.com:publiccodenet/standard.git
 * [new tag]         0.2.0 -> 0.2.0
```

**Comment by ericherman** [comment]
I generated the `standard-for-public-code-0.2.0.pdf` and attached it to https://github.com/publiccodenet/standard/releases/tag/0.2.0

**Pull request merged**
